feat: pipeline file examples

roleyfoley opened the pull request:
A collection of example pipelines that implement the Hamlet CI/CD Pipeline processes

roleyfoley committed:
feat: pipeline file examples

ml019 commented:
Does automatic deploy need to show empty steps for build and test to show where they fit?

Also how to use change sets to potentially have multiple units in one repo?

I['m assuming things in <> need replacement? Maybe a comment that effect at the top of each file?

roleyfoley committed:
feat: build pipeline examples and docs

roleyfoley commented:
> Does automatic deploy need to show empty steps for build and test to show where they fit?
> 
> Also how to use change sets to potentially have multiple units in one repo?
> 
> I['m assuming things in <> need replacement? Maybe a comment that effect at the top of each file?

Added two build jobs a simple one and a mutliple units build to show the build process triggering the other jobs 
Added docs file to explain how they should be used

ml019 commented:
Perrrrrrfect.
